app/assets/javascripts/plugins/ecommerce/admin_product.js.coffee

```diff
@@ -3,6 +3,11 @@ $ ->
   variation_id = 1
   product_variations = form.find('#product_variations')
   form.find('.content-frame-body > .c-field-group:last').after(product_variations.removeClass('hidden'))
+  SERVICE_PRODUCT = "service product"
```

Comment:
Please use one-word strings instead for when used as variables for comparisons (instead of "service product" you could have used "service_product").